### ISSUE TITLE
NVDA Crash With Jump to Line in Log

### ISSUE THREAD

**Issue by fisher729** [lukaszgo1/percentageChecker#1]
Hi.

STR.
1. Open NVDA log with NVDA+F1.
2. Press NVDA+Shift+J to activate Jump to Line dialog.
Expected: Jump to Line dialog should appear
Actual: NVDA crashes, and an Alt+Tab and Ctrl+Alt+N is required.
Other info: Using latest dev release (1.3) and Master with same results.
Technical: I don't know if jumping to line is not supported in the log. If not, this should be blocked rather than cause NVDA to crash.

**Comment by lukaszgo1** [comment]
Thanks for your report - I can certainly  reproduce this issue and I'll investigate a fix. Out of interest in which places functionality of jumping to a given line is useful to you? I've just ported it from the Oriol's code but I don't see much use for it personally as most programs in which jumping to a line can be useful contains a function to do it natively.

**Comment by fisher729** [comment]
Hi

Thanks. I mostly use it in Notepad, when reviewing code and am told to
go to line X. It is also useful in browsers as well, but not as much.

**Comment by lukaszgo1** [comment]
Any reason not to simply use Ctrl+g in Notepad?

**Comment by fisher729** [comment]
It doesn't work for me. When I press the shortcut, nothing happens,
and it is unavailable from the menus.

I still think it's a very useful feature, in terms of how you have
configured it, as it tells you how many lines are in the text in other
types of documents, such as browse mode.

**Comment by lukaszgo1** [comment]
Hi @fisher729 I've worked around it by disabling jumping to line in NVDA's edit  fields. I'm not really satisfied with this solution, but the crash is caused by the bug in NVDA which I cannot fix inside the add-on. I'll certainly report this against NVDA and if tthe bug is fixed I'll disable jumping to line only for the affected versions of NVDA. With regard to Ctrl+g in Notepad you need to disable word wrap in the format menu  to have it available.

**Comment by fisher729** [comment]
Hi.

Ok, nice workaround. I will also try that.
